src/spu/interface.h

```diff
@@ -167,6 +168,8 @@ class impl final : public SPUInterface {
     REVERBInfo rvb;
 
     uint32_t dwNoiseVal = 1;  // global noise generator
+    uint32_t dwNoiseClock = 0;  // global noise generator
```

Comment:
These naming schemes are deprecated. We should try to avoid adding more. This should be called `m_noiseClock` and `m_noiseCount`.